src/helpers/misc.js

```diff
@@ -82,14 +82,28 @@ const asyncTaskCreator = (func, params) => {
   };
 };
 
+/**
+ * This function reads the registry - it a fix for require issues.
+ * @param {String} [path] - path to registry
+ */
+const getRegistry = (path) => {
+  try {
+    return JSON.parse(fs.readFileSync(path, 'utf8'));
```

Comment:
This is only happy path scenario, what if it will throw an error?
No registry should equal stopping generation or fallback to local - but first is better IMO